apisix/plugins/response-rewrite.lua

```diff
@@ -115,6 +152,23 @@ function _M.check_schema(conf)
         end
     end
 
+    if conf.filters then
+        for _, filter in ipairs(conf.filters) do
+            for field, value in pairs(filter) do
+                if type(field) ~= 'string' then
```

Comment:
We don't need to check the type of field. It doesn't do any harm as we only consume known fields.

Reply:
Ok, had remove it.